pkg/arvo/sys/lull.hoon

```diff
@@ -1069,14 +1069,23 @@
   ::::                                                  ::  (1d2)
     ::
   +$  blew  [p=@ud q=@ud]                               ::  columns rows
-  +$  belt                                              ::  old belt
+  +$  belt                                              ::  client input
+    $%  bolt                                            ::  simple input
```

Comment:
Curiously, `$?` works fine in #4463, but gives `fish-loop` here. `$%` works somewhat unexpectedly, considering the `@c` case in `bolt`. Probably benign shenanigans afoot.

Reply:
...and it turns out this actually explodes over as soon as you try to actually use a `@c` `bolt`.

Reply:
The fish-loop trace points right here?

Reply:
The entire `$?` (up to the `==`), yes.